keyboards/smk60/smk60.c

```diff
@@ -0,0 +1,5 @@
+/**
+ * smk60.c
+ */
+
+#include QMK_KEYBOARD_H
```

Comment:
```suggestion
#include "smk60.h"
```

Reply:
Is this the current recommend style? last time seems QMK_KEYBOARD_H was preferred.

Reply:
Yes, but not in this file. `#include QMK_KEYBOARD_H` goes in `keymap.c`.

Reply:
thanks, they were fixed